STYLEGUIDE.md

```diff
@@ -130,6 +131,33 @@ export default Ember.Component.extend({
 });
 ```
 
+### Documentation
+
+Follow the syntax outlined on the [YUIDoc Syntax Page](http://yui.github.io/yuidoc/syntax/index.html). As YUIDoc was built for javascript, not for ember specifically, we have defined our own paradigm.
+
+* `Components/Services/Routes/Models` will each be a module. All extensions of them should define which `@module` it belongs to.
+* All extensions of these modules are defined as `@class`
+* Actions are defined as `@method`, but should be specified that they are actions in the description.
```

Comment:
This is a bit unfortunate that actions can't be defined separately. Let me know if there are any suggestions here.